server.elm

```diff
@@ -3328,19 +3416,19 @@ editionscript = """
         interactionDiv.append(el("h1", {}, "Insert"));
         interactionDiv.append(el("div", {id: "insertionPlace"}, [
           clickedElem.tagName === "BODY" || clickedElem.tagName === "HTML" || clickedElem.tagName === "HEAD" ? undefined :
-            el("span", {}, [
+            el("span", {id: "insertOption"}, [
```

Comment:
ids should be unique. across the entire document. Use `{class: "insertOption"}` instead